### ISSUE TITLE
Railcraft crashes if fluid blocks are disabled

### ISSUE THREAD

**Issue by GreatOrator** [Railcraft/Railcraft#1475]
GC Edit: occurs regardless of other mods, crash is result if any fluid block is disabled

## Description
The primary issue is that railcraft appears to be overwriting the immersive engineering creosote as null:

[07:57:28] [Client thread/WARN] [FML]: A mod has attempted to assign Block Block{null} to the Fluid 'creosote' but this Fluid has already been linked to the Block Block{immersiveengineering:fluidCreosote}. You may have duplicate Fluid Blocks as a result. It *may* be possible to configure your mods to avoid this.

The crash happens if I try to disable creosote in the primary railcraft.cfg or the creosote block in block.cfg.

I found there were multiple reports between RC and IE about this issue (the missing texture/lack of functionality) yet no resolution.  Since this report above is happening AFTER IE loads it's fluid and is caused by RC I am starting here.

But, rather than simply say it is IE's fault, can the two of you attempt to work together to find the cause of the missing fluid/texture/functionality or find a way to disable creosote within railcraft without crashing the game on load?

@blusunrize

## Base information
* Minecraft version: 1.10.2
* Minecraft Forge version: 12.18.3.2511

## Crash report
Client: 
```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Railcraft (railcraft)
Caused by: java.lang.NullPointerException
	at mods.railcraft.common.fluids.RailcraftFluids.initBlock(RailcraftFluids.java:156)
	at mods.railcraft.common.fluids.RailcraftFluids.init(RailcraftFluids.java:105)
	at mods.railcraft.common.fluids.RailcraftFluids.preInitFluids(RailcraftFluids.java:93)
	at mods.railcraft.common.modules.ModuleCore$1.construction(ModuleCore.java:95)
	at mods.railcraft.common.modules.RailcraftModulePayload$BaseModuleEventHandler.construction(RailcraftModulePayload.java:72)
	at mods.railcraft.common.modules.RailcraftModuleManager$Stage$1.passToModule(RailcraftModuleManager.java:270)
	at mods.railcraft.common.modules.RailcraftModuleManager.processStage(RailcraftModuleManager.java:227)
	at mods.railcraft.common.modules.RailcraftModuleManager.preInit(RailcraftModuleManager.java:176)
	at mods.railcraft.common.core.Railcraft.preInit(Railcraft.java:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:616)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:624)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:259)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:439)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:351)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```
Server:
```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Railcraft (railcraft)
Caused by: java.lang.NullPointerException
	at mods.railcraft.common.fluids.RailcraftFluids.initBlock(RailcraftFluids.java:156)
	at mods.railcraft.common.fluids.RailcraftFluids.init(RailcraftFluids.java:105)
	at mods.railcraft.common.fluids.RailcraftFluids.preInitFluids(RailcraftFluids.java:93)
	at mods.railcraft.common.modules.ModuleCore$1.construction(ModuleCore.java:95)
	at mods.railcraft.common.modules.RailcraftModulePayload$BaseModuleEventHandler.construction(RailcraftModulePayload.java:72)
	at mods.railcraft.common.modules.RailcraftModuleManager$Stage$1.passToModule(RailcraftModuleManager.java:270)
	at mods.railcraft.common.modules.RailcraftModuleManager.processStage(RailcraftModuleManager.java:227)
	at mods.railcraft.common.modules.RailcraftModuleManager.preInit(RailcraftModuleManager.java:176)
	at mods.railcraft.common.core.Railcraft.preInit(Railcraft.java:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:616)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:624)
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99)
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:328)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:121)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431)
	at java.lang.Thread.run(Thread.java:748) 
```

**Comment by Generalcamo** [comment]
Which version of Railcraft are you using?

**Comment by GreatOrator** [comment]
Railcraft_1.10.2-10.3.1

and 

ImmersiveEngineering-0.10-61

**Comment by Generalcamo** [comment]
Completely irrelevant to IE, renaming issue accordingly. This issue occurs if any fluid block is disabled.

**Comment by GreatOrator** [comment]
Cool,

As long as the issue is identified, I just chose that title because of how I came across it :D  thank you

**Comment by liach** [comment]
Fixed by 779a30a
